app/BUILD.bazel

```
@@ -196,6 +185,17 @@ ALTERED_VIEW_MODELS = [
     "src/main/java/org/oppia/app/topic/info/TopicInfoViewModel.kt",
 ]
 
+[VIEW_MODEL_FILES.append("update_" + altered_view_models[0:-3])
+for altered_view_models in ALTERED_VIEW_MODELS]
```

```suggestion
] + [
    VIEW_MODEL_FILES.append("update_" + altered_view_models[0:-3])
    for altered_view_models in ALTERED_VIEW_MODELS
]
```

Nit on formatting. We should keep concatenated lists close together.

---

Adding this statement 
```
 [
    VIEW_MODEL_FILES.append("update_" + altered_view_models[0:-3])
    for altered_view_models in ALTERED_VIEW_MODELS
] 
```
to ALTERED_VIEW_MODELS isn't correct here. It triggers a "ALTERED_VIEW_MODELS is referenced before assignement" error. This line isn't creating a list that is going to be later concatenated with another list - rather it is appending each element in ALTERED_VIEW_MODELS to VIEW_MODEL_FILES.

---

Ah, I had misread this. Thanks for clarifying!

---

Aha, actually what I meant was:

```suggestion
] + [
    VIEW_MODEL_FILES.append("update_" + altered_view_models[0:-3])
    for altered_view_models in ALTERED_VIEW_MODELS
]
```

Nit on formatting. We should keep concatenated lists close together.

---

Ah I see! This makes sense - fixed!